pkg/ui/workspaces/cluster-ui/webpack.config.js

```diff
@@ -168,6 +169,15 @@ module.exports = (env, argv) => {
         /node_modules\/antd\/lib\/style\/index\.less/,
         path.resolve(__dirname, "src/core/antd-patch.less"),
       ),
+
+      // Use MomentTimezoneDataPlugin to remove timezone data that we don't need.
+      new MomentTimezoneDataPlugin({
+        matchZones: ['Etc/UTC', 'America/New_York'],
+        startYear: 2021,
```

Comment:
I can't find an answer reading through the source yet — how many years does this include if we only specify `startYear`?

Reply:
`currentYear + 10` is probably safe and not super heavy! It gives us quite a bit of buffer beyond the 3-version support window.